src/node_crypto.h

```diff
@@ -76,6 +76,8 @@ class SecureContext : public BaseObject {
 
   static void Initialize(Environment* env, v8::Local<v8::Object> target);
 
+  static void UseExtraCaCerts(const char* file);
```

Comment:
It's kind of odd that this is a (static) method but `LoadExtraCaCerts()` is not.  Neither manipulate SecureContext state so they probably shouldn't be methods, static or not.